ui/lib/pki/addon/components/page/pki-overview.hbs

```diff
@@ -29,7 +29,7 @@
   <OverviewCard @cardTitle="Issue certificate" @subText="Begin issuing a certificate by choosing a role.">
     <div class="has-top-margin-m is-flex">
       <SearchSelect
-        class="is-flex-grow-1"
+        class="is-marginless is-flex-grow-1"
```

Comment:
The other day I pinged you about another approach for this in slack - just messaged you about it

Reply:
Just messaged. It solves. the issue of your comment below, but also makes the button smaller. I think it looks okay, but wanted to check with you first.